mathics/core/expression.py

```diff
@@ -599,7 +557,19 @@ def do_format(self, evaluation, form):
 
     @property
     def elements(self):
-        return self._elements
+        return self.__elements
```

Comment:
I didn't understant why change `_elements` to `__elements`.

Reply:
Because there are several places in which `Expression._elements` is set, without calling `_build_elements`. With this change, each time `Expression._elements` is set, now the setter is called.